internal/app/starter/master_linux.go

```diff
@@ -65,9 +65,6 @@ func Master(rpcSocket, masterSocket int, isInstance bool, containerPid int, engi
 		}); ok {
 			n, err := conn.Read(data)
 			if (err != nil && err != io.EOF) || n == 0 || data[0] == 'f' {
-				if isInstance && os.Getppid() == ppid {
```

Comment:
I'm assuming that we remove this because we're now only sending a signal if master succeeds (`SIGUSR1`) and otherwise issuing a non-zero exit code to signal to `init()` C function that master process failed. If that's the case, then shouldn't we not just `return` from here but actually either `exit(1)` or `sylog.Fatalf()` or similar?

Reply:
Or, similar to Line 85 in the new code do `fatalChan <- fmt.Errorf(...)`?

Reply:
I will look